### PR TITLE
Update pubspec.yaml

### PR DESCRIPTION
**Pull request by muhammadhuzaifakhan1133**
I upgrade http package to the latest version.

### CODE REVIEW

**Comment by muhammadhuzaifakhan1133** [comment]
I am getting a version-solving error whenever I try to add 'upgrade' package to dependencies. The reason is that I have also a http package of the latest version (1.1.0), but your package depends on old versions of http package, so that's why I got a version-solving error. The error code is attached below:

Because every version of 'upgrade' depends on http ^0.13.5 and agora_ui_kit depends on http ^1.1.0, upgrade is forbidden.
So, because agora_ui_kit depends on upgrade any, version solving failed.

File: pubspec.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ dependencies:
   html: ^0.15.3
 
   # From Dart Team: A composable, Future-based library for making HTTP requests.
-  http: '>=0.13.6 <2.0.0'
+  http: ^1.1.0
```
Comment (larryaasen) [inline pubspec.yaml:21]: This would break compatibility with versions of 0.13.6 of http.

Comment (larryaasen) [reply pubspec.yaml:21]: The upgrader package already supports http 1.1.0

**Comment by larryaasen** [comment]
@muhammadhuzaifakhan1133 Is this still an issue for you?

**Pull request closed without merging**